scripts/bert/launcher_mx_bert.py

```diff
@@ -0,0 +1,77 @@
+import os
```

Comment:
Should we have this launcher script as python script or jupyter notebook script?